app/jobs/doi_removal_job.rb

```diff
@@ -0,0 +1,7 @@
+class DOIRemovalJob < ActiveJob::Base
```

Comment:
I pulled in resque and using activejob API (comes with rails 4.1+), instead of hooking into Sufia's job queue like we have done with app/jobs/batch_update_job.rb. Why?

Sufia's job approach is a custom api on top of resque which seems to be entirely replaced in Sufia 7+. Why not follow this approach? Well, for one they are marshalling the entire job in the queue which loses visibility and information about the job. As a result they are naming their jobs as queues, which isn't the purpose of a queue. For example:
```
Sufia::Resque::MarshaledJob    ["BAhjEkRPSU1pbnRpbmdKb2I=\n"]
```
vs
```
ActiveJob::QueueAdapters::ResqueAdapter::JobWrapper    [{"job_class"=>"DOIRemovalJob", "job_id"=>"e5f88c8a-3f28-435e-902d-a340ee36194d", "queue_name"=>"default", "arguments"=>["doi:10.5072/FK2ZW1FW18"], "locale"=>"en"}]
```
Also they are not following the standard resque API (for example using a run method instead of perform method which seems to have no documentation about these changes, etc.). With Sufia 7.0+, they have removed this approach entirely and simply uses ActiveJob API which is great! So for sake of future proofing this and doing this better, I decided to use ActiveJob API instead.